hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManagerV2.java

```diff
@@ -84,6 +85,43 @@ ContainerInfo getContainer(ContainerID containerID)
    */
   List<ContainerInfo> getContainers(LifeCycleState state);
 
+
+  /**
+   * Returns containers under certain conditions.
+   * Search container IDs from start ID(exclusive),
+   * The max size of the searching range cannot exceed the
+   * value of count.
+   *
+   * @param startContainerID start containerID, >=0,
+   * start searching at the head if 0.
+   * @param count count must be >= 0
+   *              Usually the count will be replace with a very big
+   *              value instead of being unlimited in case the db is very big.
+   * @param state Container of this state will be returned. Can be null.
+   *
+   * @return a list of container.
+   * @throws IOException
+   */
+  List<ContainerInfo> listContainer(ContainerID startContainerID, int count,
```

Comment:
Since we already have ` List<ContainerInfo> getContainers(LifeCycleState state);`, the two interface is not needed, they are duplicate with `getContainers`.